From: Julian Taylor <jtaylor.debian@googlemail.com>
Date: Wed, 19 Oct 2011 18:37:25 +0200
Subject: use LightBG colorscheme

by default Linux scheme leads to invisible text when console background
is light LightBG is at least readable with dark background
Forwarded: not-needed
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/ipython/+bug/241535

---
 IPython/core/interactiveshell.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/IPython/core/interactiveshell.py
+++ b/IPython/core/interactiveshell.py
@@ -125,7 +125,7 @@ def get_default_colors():
     elif os.name=='nt':
         return 'Linux'
     else:
-        return 'Linux'
+        return 'LightBG'
 
 
 class SeparateUnicode(Unicode):
